### PR TITLE
Adds missing method .compare in SortedOps

### PR DESCRIPTION
**Pull request by Prithvirajbilla**
Fixes scala/bug#10980

### CODE REVIEW

File: src/library/scala/collection/SortedOps.scala [hunk 1]
```diff
@@ -12,6 +12,10 @@ trait SortedOps[A, +C] {
   /** Returns the last key of the collection. */
   def lastKey: A
 
+  /** Comparison function that orders keys. */
+  @deprecated("Use Ordering.compare instead.", "2.13.0")
+  @inline def compare(k0: A, k1: A): Int = ordering.compare(k0, k1)
```
Comment (julienrf) [inline src/library/scala/collection/SortedOps.scala]: We should also mark it `@deprecatedOverriding`.

Comment (Prithvirajbilla) [reply src/library/scala/collection/SortedOps.scala]: updated. thanks

File: src/library/scala/collection/SortedOps.scala [hunk 2]
```diff
@@ -12,6 +12,11 @@ trait SortedOps[A, +C] {
   /** Returns the last key of the collection. */
   def lastKey: A
 
+  /** Comparison function that orders keys. */
+  @deprecated("Use Ordering.compare instead.", "2.13.0")
```
Comment (julienrf) [inline src/library/scala/collection/SortedOps.scala]: Actually the suggestion is not correct, it should be `ordering.compare` instead (lower case).

Comment (Prithvirajbilla) [reply src/library/scala/collection/SortedOps.scala]: updated. thanks

**Comment by julienrf** [comment]
Thanks @Prithvirajbilla !

**Pull request merged**